MiniApp/Classes/JavascriptBridge/MiniAppJavaScriptEnumerations.swift

```diff
@@ -10,4 +11,28 @@ enum JavaScriptExecResult: String {
 enum MiniAppJavaScriptError: String {
     case internalError
     case unexpectedMessageFormat
+    case invalidPermissionType
+}
+
+public enum MiniAppPermissionType: String {
+    case location
+}
+
+public enum MiniAppPermissionResult: Error {
+    case denied
```

Comment:
Should model the same structure as adapted by the OS. Including One-Time Use.

Reply:
Yes, it is same as CLAuthorizationStatus structure